Site/SiteConcentrateFileFinder.php

```diff
@@ -53,7 +53,8 @@ protected function getComposerDataFiles()
 		$base_path = dirname($www_path).DIRECTORY_SEPARATOR.'vendor';
 		if (is_dir($base_path)) {
 			$base_dir = dir($base_path);
-			while (false !== ($vendor_name = $base_dir->read())) {
+			$vendor_name = $base_dir->read();
+			while ($vendor_name !== false) {
```

Comment:
This will break stuff. The end of the loop needs to do `$vendor_name = $base_dir->read()`